Clarify that Gitea is optional. (#145)
opendatahub-io/ai-edge: README.md

  * Edit contents of [thanos-secret](acm/odh-core/acm-observability/secrets/thanos.yaml) file.

### Gitea in cluster git server for GitOps workflow

You can use your own git server instead of GitHub.

> For example, the [gitea-operator](https://github.com/rhpds/gitea-operator) can be used to manage the Gitea server installation in the cluster.  This will simplify the setup of Gitea so that we can create a minimal [Gitea](https://github.com/rhpds/gitea-operator#migrating-repositories-for-created-users) CR to configure and install the Gitea server.

Reviewer: ```suggestion
For example, the [gitea-operator](https://github.com/rhpds/gitea-operator) can be used to manage the Gitea server installation in the cluster. It will simplify the setup so that you can create a minimal [Gitea](https://github.com/rhpds/gitea-operator#migrating-repositories-for-created-users) CR to configure and install the Gitea server.
```

Author: Applied in faf98ae9c7be8c2051f1c1b88ec344b21831381e.
